config/telink/chip-module/Kconfig.defaults

```diff
@@ -104,13 +104,13 @@ config NET_PKT_TX_COUNT
     default 8
 
 config NET_BUF_RX_COUNT
+    default 12 if SOC_RISCV_TELINK_TL321X || SOC_SERIES_RISCV_TELINK_B9X_RETENTION
     default 16 if PM
```

Comment:
```
    default 12 if SOC_RISCV_TELINK_TL321X || SOC_SERIES_RISCV_TELINK_B9X_RETENTION || PM
    default 32
```
    I think change to these should be better , if we open PM , which means is ICD mode , the buffer cnt shoud be the same , if we change the PM mode to 32 , some compile check may fail .

Reply:
PM config is enabled for RETENTION boards and for suspend mode.
For RETENTION you used 12
not sure if for suspend we need to decrease value
Anyway for config NET_BUF_RX_COUNT we may use two variants instead of tree

Reply:
agree , pm is binding to the retention setting , we will not use the suspend mode to do mass product , In our product , there will be two mode , normal mode , retention mode , the first setting have already cover all of pm mode .